src/lightning/fabric/plugins/precision/bitsandbytes.py

```diff
@@ -116,7 +116,10 @@ def convert_module(self, module: torch.nn.Module) -> torch.nn.Module:
                 m.compute_type_is_set = False
         return module
 
-    def init_context(self) -> ContextManager:
+    def tensor_init_context(self) -> ContextManager:
```

Comment:
L131 could call this

This suggestion would also apply to all other plugins, I won't repeat myself in all of them